### PR TITLE
UI modifications

### PR DESCRIPTION
**Pull request by steved**
* fix for #531 
* unify stop buttons so jobs and deploys are relatively similar

### CODE REVIEW

File: app/controllers/jobs_controller.rb [hunk 1]
```diff
@@ -50,13 +50,13 @@ def enabled
...
+    if @job.can_be_stopped_by?(current_user)
       @job.stop!
-
-      head :ok
     else
-      head :forbidden
+      flash[:error] = "You do not have privileges to stop this job."
     end
+
+    redirect_to [@project, @job]
```
Comment (jonmoter) [inline app/controllers/jobs_controller.rb]: Does the `JobsController#destroy` method ever get called via AJAX? If so, a `:forbidden` response makes more sense than a redirect.

Comment (steved) [inline app/controllers/jobs_controller.rb]: Not anymore, it used to be

File: app/views/jobs/_output.html.erb [hunk 2]
```diff
@@ -7,12 +7,11 @@
...
-      <div class="pull-right only-finished">
-        <button class="btn btn-default" id="output-grow-toggle">Expand</button>
-        <%= link_to "Download log", project_job_path(@project, job, format: :text), class: "btn btn-primary" %>
-      </div>
+      <button class="btn btn-default only-finished" id="output-grow-toggle">Expand</button>
+      <%= link_to "Download log", project_job_path(@project, job, format: :text), class: "btn btn-primary only-finished" %>
+
       <% if job.can_be_stopped_by?(current_user) %>
-        <%= stop_button(class: "btn btn-danger pull-right only-active") %>
+        <%= stop_button(deploy: job, class: "btn btn-danger only-active") %>
```
Comment (jonmoter) [inline app/views/jobs/_output.html.erb]: That's a little weird to pass in a job when it appears to be expecting an instance of `Deploy`, but I see what you're doing, and can't think of a better name for the parameter off the top of my head.

**Comment by jonmoter** [comment]
:+1:

**Pull request merged**